scripts/install.sh

```diff
@@ -1,490 +1,57 @@
     exit 1
-  fi
-
-  ensure mkdir -p "$_tempdir"
-
-  create_install_dir "$_requested_version"
-  local _installdir=$RETVAL
-  assert_nz "$_installdir" "installdir"
-
-  download "$_resolved_version" "$_arch" "$_installdir" "$_tempdir"
-
-  say "installed snforge and sncast to ${_installdir}"
-
-  create_symlinks "$_installdir"
-  local _retval=$?
-
-  echo
-  if echo ":$PATH:" | grep -q ":${LOCAL_BIN}:"; then
-    echo "Starknet Foundry has been successfully installed and should be already available in your PATH."
-    echo "Run 'snforge --version' and 'sncast --version' to verify your installation. Happy coding!"
-  else
-    if [ $_do_modify_path -eq 1 ]; then
-      add_local_bin_to_path
-      _retval=$?
-    else
-      echo "Skipping PATH modification, please manually add '${LOCAL_BIN_ESCAPED}' to your PATH."
-    fi
-
-    echo "Then, run 'snforge --version' and 'sncast --version' to verify your installation. Happy coding!"
-  fi
-
-  ignore rm -rf "$_tempdir"
-  return "$_retval"
-}
-
-# This function has been copied verbatim from rustup install script.
-check_proc() {
-    # Check for /proc by looking for the /proc/self/exe link
-    # This is only run on Linux
-    if ! test -L /proc/self/exe ; then
-        err "fatal: Unable to find /proc/self/exe.  Is /proc mounted?  Installation cannot proceed without /proc."
-    fi
-}
-
-# This function has been copied verbatim from rustup install script.
-get_bitness() {
-    need_cmd head
-    # Architecture detection without dependencies beyond coreutils.
-    # ELF files start out "\x7fELF", and the following byte is
-    #   0x01 for 32-bit and
-    #   0x02 for 64-bit.
-    # The printf builtin on some shells like dash only supports octal
-    # escape sequences, so we use those.
-    local _current_exe_head
-    _current_exe_head=$(head -c 5 /proc/self/exe )
-    if [ "$_current_exe_head" = "$(printf '\177ELF\001')" ]; then
-        echo 32
-    elif [ "$_current_exe_head" = "$(printf '\177ELF\002')" ]; then
-        echo 64
-    else
-        err "unknown platform bitness"
-    fi
-}
-
-say() {
-  printf 'starknet-foundry-install: %s\n' "$1"
-}
-
-err() {
-  say "$1" >&2
-  exit 1
-}
-
-need_cmd() {
-  if ! check_cmd "$1"; then
-    err "need '$1' (command not found)"
-  fi
-}
-
-check_cmd() {
-  command -v "$1" >/dev/null 2>&1
-}
-
-assert_nz() {
-  if [ -z "$1" ]; then err "assert_nz $2"; fi
-}
-
-# Run a command that should never fail.
-# If the command fails execution will immediately terminate with an error showing the failing command.
-ensure() {
-  if ! "$@"; then err "command failed: $*"; fi
-}
+fi
 
-# This is just for indicating that commands' results are being intentionally ignored.
-# Usually, because it's being executed as part of error handling.
-ignore() {
-  "$@"
-}
+# Create the ${HOME}/.local/bin bin directory and snfoundryup binary if it doesn't exist.
+mkdir -p ${LOCAL_BIN}
+curl -# -L ${SNFOUNDRYUP_URL} -o ${SNFOUNDRYUP_PATH}
+chmod +x ${SNFOUNDRYUP_PATH}
```

Comment:
I know these are quite safe, but let's quote the variable names just to preserve good practices :+1: